<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TrainingDisplay</class>
 <widget class="QDialog" name="TrainingDisplay">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>730</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>730</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QGroupBox" name="grpBoxLearningPlan">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>450</height>
      </size>
     </property>
     <property name="title">
      <string>Learning Plan</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="1" column="0">
       <widget class="QLabel" name="lblNoTraining">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>                                                                                 No Trainings to Display</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTableWidget" name="tblLearningPlan"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="grpBoxLearningHistory">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>220</height>
      </size>
     </property>
     <property name="title">
      <string>Learning History</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="1" column="0">
       <widget class="QLabel" name="lblNoLearningRecord">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>                                                                                 No Learning Records</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QTableWidget" name="tblLearningHistory"/>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
